### ISSUE TITLE
Slider reaches 100% before loading completes

### ISSUE THREAD

**Issue by lolmaus** [jerel/ember-cli-loading-slider#25]
Hi.

As it's impossible to predict loading times, a good progress slider never reaches 100%. Instead, it would gradually slow down and never reach 100%, just like [Achilles and his tortoise](https://en.wikipedia.org/wiki/Zeno%27s_paradoxes#Achilles_and_the_tortoise):

<img alt src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/66/Zeno_Achilles_Paradox.png/480px-Zeno_Achilles_Paradox.png" width="240" height="240">

Subj reaches 100% pretty quickly, while the app is still loading, effectively defeating its purpose.

**Comment by jerel** [comment]
Fair point. I'd be happy to review a PR that uses back-off to keep it from finishing prematurely
